tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/Keramelizer.scala

```diff
@@ -64,6 +64,39 @@ class Keramelizer(gen: UniqueNameGenerator, tracker: TransformationTracker)
       tla
         .not(tla.in(x, setX) ? "b")
         .typed(Map("b" -> BoolT1()), "b")
+    // rewrite POWSET A \subseteq POWSET B
+    // into A \subseteq B
+    case OperEx(TlaSetOper.subseteq, OperEx(TlaSetOper.powerset, setX), OperEx(TlaSetOper.powerset, setY)) =>
+      transform(
+          tla
+            .subseteq(setX, setY)
+            .typed(Map("b" -> BoolT1()), "b")
+      )
+
+    // rewrite A \subseteq POWSET B
+    // into \A S \in A: S \subseteq B
+    case OperEx(TlaSetOper.subseteq, setX, OperEx(TlaSetOper.powerset, setY)) =>
+      val elemType = getElemType(setY)
+      val tempName = gen.newName()
+      transform(
+          tla
+            .forall(tla.name(tempName) ? "s", setX, tla.subseteq(tla.name(tempName) ? "s", setY) ? "b")
+            .typed(Map("b" -> BoolT1(), "e" -> elemType, "s" -> SetT1(elemType)), "s")
+      )
+
+    // rewrite A \subseteq B
+    // into \A a \in A: a \in B
+    case OperEx(TlaSetOper.subseteq, setX, setY) =>
+      replaceSubsetWithQuantification(setX, setY)
```

Comment:
If you're going to invoke `replaceSubsetWithQuantification` exactly once, you might as well just put the body here directly.